src/render/mod.rs

```diff
@@ -7,6 +7,7 @@ use std::cmp::max;
 pub mod image;
 pub mod string;
 pub mod svg;
+pub mod utf8;
```

Comment:
Let's rename the module to "unicode". 

UTF-8 is a string encoding but the output doesn't rely on properties of UTF-8.

Reply:
Maybe rename Unicode1x2 to smth else as well? Lint doesn't like repetitions.